cmd/kube-apiserver/app/options/options.go

```diff
@@ -33,6 +33,8 @@ type APIServer struct {
 	*genericapiserver.ServerRunOptions
 	AllowPrivileged             bool
 	EventTTL                    time.Duration
+	KeystoneConfig              string
+	KeystoneAuthMode            string
```

Comment:
these two fields have been move to `genericapiserver.ServerRunOptions`, please remove them.